modules.d/01systemd-repart/module-setup.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+# This file is part of dracut.
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Prerequisite check(s) for module.
+check() {
+
+    # If the binary(s) requirements are not fulfilled
+    # return 1 to not include the binary.
+    require_binaries systemd-repart || return 1
+
+    # If the module dependency requirements are not fulfilled
+    # return 1 to not include the required module(s).
+    if ! dracut_module_included "systemd"; then
+        derror "systemd-coredump needs systemd in the initramfs."
```

Comment:
this is probably the wrong error message

Reply:
Fixed.